src/app/components/shell/router.js

```diff
@@ -146,7 +156,7 @@ function MainRoutes() {
                 <Route path="/details/" element={<Navigate to="/subjects" replace />} />
                 <Route path="/books/:title" element={<RedirectToCanonicalDetailsPage />} />
                 <Route path="/textbooks/:title" element={<RedirectToCanonicalDetailsPage />} />
-                <Route path="/subjects-preview/*" element={<ImportedPage name="subjects" />} />
+                <Route path="/subjects/*" element={<ImportedPage name="subjects" />} />
```

Comment:
the subjects route exists already somewhere doesn't it? i don't see it but it must...

Reply:
It was using the `/:name/*` path below, but since that's changing, it needs its own entry.